<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsWorkflowStepEditorDlgClass</class>
 <widget class="QDialog" name="CwmsWorkflowStepEditorDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arbeitsschritt bearbeiten</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="m_pqlArticleType">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(232, 232, 232, 255));</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Arbeitsschritt bearbeiten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_pqwWorkflowStep">
      <attribute name="title">
       <string>Arbeitsschritt</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="m_pqleName"/>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Beschreibung</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="m_pqteDescription"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Verantwortungsbereich</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="3">
        <widget class="CwmsObjectChoiceComboBox" name="m_pCwmsTeam"/>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Statusänderung</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="3">
        <widget class="QComboBox" name="m_pqcbState">
         <item>
          <property name="text">
           <string>Offen</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In Prüfung</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In Arbeit</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Erledigt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nicht berechtigt</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Geschlossen</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Maximale Dauer in Tagen</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="3">
        <widget class="QSpinBox" name="m_pqsbMaxDuration">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="3">
        <widget class="QComboBox" name="m_pqcbType">
         <item>
          <property name="text">
           <string>Start</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Laufend</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ende</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Typ</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Abarbeitungsschritt-Nr.</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="m_pqlePosition">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="m_pqleSubWorkflow">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>kein Prozess gewählt</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="m_pqpbAddSubProcess">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../WMSadmin/resources/emresources.qrc">
           <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QPushButton" name="m_pqpbRemoveSubProcess">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../WMSadmin/resources/emresources.qrc">
           <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Subprozess</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pqwChangingValues">
      <attribute name="title">
       <string>Wertänderungen</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Wertänderungen beim Eintritt in diesen Prozessschritt</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="m_pqtwValueChanges">
            <column>
             <property name="text">
              <string>Member</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wert</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="m_pqpbAddMemberValue">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbRemoveMemberValue">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pqwHiddenValues">
      <attribute name="title">
       <string>Versteckte Felder</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Members</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QListWidget" name="m_pqlwHiddenMembers"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="m_pqpbAddHiddenMember">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbRemoveHiddenMember">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pqwReadOnlyValues">
      <attribute name="title">
       <string>Felder mit Leseberechtigung</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Members</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QListWidget" name="m_pqlwReadOnlyMembers"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="m_pqpbAddReadOnlyMember">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/new24.png</normaloff>:/icons/new24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pqpbRemoveReadOnlyMember">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../WMSadmin/resources/emresources.qrc">
             <normaloff>:/icons/delete24.png</normaloff>:/icons/delete24.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pqwSuccessors">
      <attribute name="title">
       <string>Nachfolger</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="CwmsObjectListEditorWidgetIf" name="m_pCwmsSuccessors" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pqwEscalation">
      <attribute name="title">
       <string>Eskalation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="CwmsObjectListEditorWidgetIf" name="m_pCwmsEscalation" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/cancel24.png</normaloff>:/icons/cancel24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbOK">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../WMSadmin/resources/emresources.qrc">
         <normaloff>:/icons/ok24.png</normaloff>:/icons/ok24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CwmsObjectListEditorWidgetIf</class>
   <extends>QWidget</extends>
   <header location="global">CwmsObjectListEditorWidgetIf.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CwmsObjectChoiceComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">cwmsobjectchoicecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../WMSadmin/resources/emresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbCancel</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbOK</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>OKClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddMemberValue</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>AddValueChangeMemberClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveMemberValue</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>RemoveValueChangeMemberClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddHiddenMember</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>AddHiddenMemberClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveHiddenMember</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>RemoveHiddenMemberClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddReadOnlyMember</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>AddReadOnlyMemberClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveReadOnlyMember</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>RemoveReadOnlyMemberClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAddSubProcess</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>AddSubProcessClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemoveSubProcess</sender>
   <signal>clicked()</signal>
   <receiver>CwmsWorkflowStepEditorDlgClass</receiver>
   <slot>RemoveSubProcessClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OKClickedSlot()</slot>
  <slot>AddValueChangeMemberClickedSlot()</slot>
  <slot>RemoveValueChangeMemberClickedSlot()</slot>
  <slot>AddHiddenMemberClickedSlot()</slot>
  <slot>RemoveHiddenMemberClickedSlot()</slot>
  <slot>AddReadOnlyMemberClickedSlot()</slot>
  <slot>RemoveReadOnlyMemberClickedSlot()</slot>
  <slot>AddSubProcessClickedSlot()</slot>
  <slot>RemoveSubProcessClickedSlot()</slot>
 </slots>
</ui>
